### PR TITLE
[t2132] optional project author in project/embed

### PR DESCRIPTION
**Pull request by secretrobotron**

### CODE REVIEW

File: cornfield/routes/index.js [hunk 1]
```diff
@@ -25,6 +25,8 @@ module.exports = function routesCtor( app, User, filter, sanitizer ) {
       var projectJSON = JSON.parse( doc.data );
       projectJSON.name = doc.name;
       projectJSON.projectID = doc._id;
+      projectJSON.author = doc.author;
+      console.log('sending', projectJSON.author);
```
Comment (dseif) [inline cornfield/routes/index.js]: remove log

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @cadecairos received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/cebf524378165a4/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @cadecairos received. Current queue size: 2

Live output at: http://botio.butyr.org:8000/bb0d702e2db78da/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/cebf524378165a4/output.txt

Total script time: 1.20 mins

Server is running on http://pr896.butyr.org/templates/basic/
Run the Test Suite: http://pr896.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/bb0d702e2db78da/output.txt

Total script time: 1.80 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @mjschranz received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/debe11b86dd1b9e/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 3

Live output at: http://botio.butyr.org:8000/12378bc2346272f/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/debe11b86dd1b9e/output.txt

Total script time: 1.67 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/12378bc2346272f/output.txt

Total script time: 0.96 mins

Server is running on http://pr896.butyr.org/templates/basic/
Run the Test Suite: http://pr896.butyr.org/test/

**Pull request merged**